db/seeds/development/submissions_seed.rb

```diff
@@ -55,88 +55,177 @@
+      guardian_two_last_name: Faker::Name.last_name,
+      guardian_two_phone_number: Faker::Base.numerify('###-###-####'),
+      guardian_two_phone_type: EnumConstants::PHONE_TYPES.sample,
+      guardian_two_relationship: EnumConstants::GUARDIAN_RELATIONSHIPS.sample,
+      health_conditions: EnumConstants::HEALTH_CONDITIONS.sample,
+      health_conditions_description: 'None',
+      home_phone: Faker::Base.numerify('###-###-####'),
+      immunizations: EnumConstants::BOOLEANS.sample,
+      insurance: EnumConstants::BOOLEANS.sample,
+      insurance_address: Faker::Address.street_address,
+      insurance_address_city: Faker::Address.city,
+      insurance_address_state: EnumConstants::STATES.sample,
+      insurance_address_zip: Faker::Address.zip,
+      insurance_id: Faker::Name.first_name,
+      insurance_phone_number: Faker::Base.numerify('###-###-####'),
+      insurance_provider: 'None',
+      insurance_other: 'None',
+      is_primary: true,
+      last_name: Faker::Name.last_name,
+      medications: 'None',
+      other_dietary_restrictions: 'None',
+      participation_guardian_consent: EnumConstants::AGREEMENTS.sample,
+      participation_guardian_name: Faker::Name.first_name,
+      participation_student_consent: EnumConstants::AGREEMENTS.sample,
+      participation_student_name: Faker::Name.first_name,
+      preferred_name: Faker::Name.first_name,
+      psychologist_consent: EnumConstants::BOOLEANS.sample,
+      psychologist_consent_name: Faker::Name.first_name,
+      risk_guardian_consent: EnumConstants::AGREEMENTS.sample,
+      risk_guardian_date: Faker::Date.between(33.days.ago, Time.zone.today),
+      risk_guardian_email: Faker::Internet.email,
+      risk_guardian_name: Faker::Name.first_name,
+      risk_guardian_relationship: EnumConstants::GUARDIAN_RELATIONSHIPS.sample,
+      risk_student_consent: EnumConstants::AGREEMENTS.sample,
+      risk_student_date: Faker::Date.between(33.days.ago, Time.zone.today),
+      risk_student_email: Faker::Internet.email,
+      risk_student_name: Faker::Name.first_name,
+      shirt_size: EnumConstants::SHIRT_SIZES.sample,
+      transportation: EnumConstants::TRANSPORTATION_OPTIONS.sample,
+      transportation_arrival_date: Faker::Date.between(33.days.ago, Time.zone.today),
+      transportation_arrival_time: Time.zone.now,
+      transportation_carrier: 'Bart',
+      transportation_consent: EnumConstants::AGREEMENTS.sample,
+      transportation_consent_name: Faker::Name.first_name,
+      transportation_departure_date: Faker::Date.between(33.days.ago, Time.zone.today),
+      transportation_departure_time: Time.zone.now,
+      transportation_name: Faker::Name.first_name,
+      transportation_number: 777,
+    )
+    student_submission.submit_submission
+    puts "Created student submission #{student_submission.id}."
+  end
 
-puts "Created student submission #{student_submission.id}."
+  student_submission = StudentSubmission.create(
```

Comment:
Want to abstract the `student_submission` create into a method that returns the created `submission` to be DRY?